manifests/params.pp

```diff
@@ -35,45 +35,51 @@
 ) {
   $subsystems = [
     'actionrunner', 'api', 'sensorcontainer',
-    'rulesengine', 'resultstracker', 'notifier',
+    'rulesengine', 'garbagecollector', 'resultstracker', 'notifier',
     'auth'
   ]
 
   $component_map = {
-    actionrunner       => 'st2actions',
-    api                => 'st2api',
-    auth               => 'st2auth',
-    notifier           => 'st2actions',
-    resultstracker     => 'st2actions',
-    rulesengine        => 'st2reactor',
-    sensorcontainer    => 'st2reactor',
-    web                => 'st2common',
-    st2actionrunner    => 'st2actions',
-    st2api             => 'st2api',
-    st2auth            => 'st2auth',
-    st2notifier        => 'st2actions',
-    st2resultstracker  => 'st2actions',
-    st2rulesengine     => 'st2reactor',
-    st2sensorcontainer => 'st2reactor',
-    st2web             => 'st2common',
```

Comment:
That's why I'm not a big fan of this "aligned indentation" - it makes diffs harder to read.